src/index.js

```diff
@@ -1,6 +1,7 @@
 import express from 'express';
 import rootRouter from './api/root';
 import docs from './docs/index';
+import { testConnection } from '../database/models';
```

Comment:
`testConnection` sounds like you are testing something. a name like `databaseConnection` would be better